cosmwasm/contracts/bindings_factory/src/contract.rs

```diff
@@ -306,5 +335,17 @@ mod query {
 
         Ok(white_list)
     }
-}
 
```

Comment:
future idea: is it possible to add a mechanism to clear all broadcasted messages?